tests/v2/validation/rbac/globalrolesv2/globalroles_v2_test.go

```diff
@@ -0,0 +1,484 @@
+package globalrolesv2
+
+import (
+	"testing"
+	"time"
+
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/rbac"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioning"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type GlobalRolesV2TestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+}
+
+func (gr *GlobalRolesV2TestSuite) TearDownSuite() {
+	gr.session.Cleanup()
+}
+
+func (gr *GlobalRolesV2TestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	gr.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(gr.T(), err)
+
+	gr.client = client
+}
+
+func (gr *GlobalRolesV2TestSuite) testValidateRBACResources(createdUser *management.User, inheritedRoles []string) (string, int) {
```

Comment:
nit, since this is a helper function, can you remove test from the name? its a little confusing when reading through the code. Seems more appropriate to just call it validateRBACResources